packages/cli/BREAKING-CHANGES.md

```diff
@@ -2,6 +2,16 @@
 
 This list shows all the versions which include breaking changes and how to upgrade.
 
+## 1.55.0
+
+### What changed?
+
+The `N8N_RESTRICT_FILE_ACCESS_TO` environment variable now also blocks access to n8n's static cache directory at `~/.cache/n8n/public`.
```

Comment:
Shouldn't this be `N8N_BLOCK_FILE_ACCESS_TO_N8N_FILES`?

Reply:
Good catch!